agent/engine/service_connect/manager_linux.go

```diff
@@ -35,8 +35,15 @@ const (
 	defaultStatusFileName     = "appnet_admin.sock"
 	defaultStatusENV          = "APPNET_AGENT_ADMIN_UDS_PATH"
 
+	agentAuthENV   = "ENVOY_ENABLE_IAM_AUTH_FOR_XDS"
+	agentAuthOff   = "0"
```

Comment:
I would suggest that we set this ENV as part of the payload sent from control plane along with other bootstrap variables.

Reply:
I don't think that makes sense. The use of UDS and lack of IAM Auth is function purely of the AppNet Agent Container and the runtime setup that this ECS Agent is doing. ANC has not control nor knowledge of how this might change over time. The way this file is setting up containers requires this flag, if it set it up differently then it might be able to use IAM_AUTH.